src/core/server/status/legacy_status.ts

```diff
@@ -106,7 +106,7 @@ const serviceStatusToHttpComponent = (
   since: string
 ): StatusComponentHttp => ({
   id: serviceName,
-  message: status.summary,
+  message: [status.summary, status.detail].filter((s) => !!s).join(' '),
```

Comment:
Opinionated: I find it hard parsing boolean operations syntax and prefer more explicit `.filter(Boolean)`